man/pstore.conf.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version='1.0'?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1+ -->
+
+<refentry id="pstore.conf" conditional="ENABLE_PSTORE"
+          xmlns:xi="http://www.w3.org/2001/XInclude">
+  <refentryinfo>
+    <title>pstore.conf</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>pstore.conf</refentrytitle>
+    <manvolnum>5</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>pstore.conf</refname>
+    <refpurpose>Pstore configuration file</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <para><filename>/etc/systemd/pstore.conf</filename></para>
+  </refsynopsisdiv>
```

Comment:
Also `<filename>/etc/systemd/pstore.conf.d/*</filename>`.

Reply:
This comment is in response to the suggestion of doing away with the linked lists in favor of a simple array. I'd like to point out that there is sorting that takes place, and the linked lists make that function tremendously simpler; if I were to use array, wouldn't I need to [re-]copy all the data in a dirent_ct each time I change the order of the list?
If there is strong opposition to the linked lists, I can do away with them, I'm not yet seeing how doing away with the lists would make this code less complicated.
I do need to investigate converting to using list.h services, as @yuwata points out.

Reply:
Per other's suggestions, I converted to using list.h macros, and in turn did away with the code I had for manipulating lists. This removed almost 100 lines all told. Perhaps this will satisfy?